### PR TITLE
Lots of changes to fetch a custom config using your embedded config closes #3106

### LINKED ISSUE #3106: CloudFlare globally blocking domain fronting

### PR DESCRIPTION
**Pull request by myleshorton**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,19 +48,13 @@ LOGGLY_TOKEN_MOBILE := d730c074-1f0a-415d-8d71-1ebf1d8bd736
...
 .PHONY: packages clean docker
 
-define package-settings
-	PACKAGED_SETTINGS="" && \
-	if [[ ! -z "$$MANOTO" ]]; then \
-		PACKAGED_SETTINGS="startupurl: https://www.facebook.com/manototv/app_128953167177144"; \
-	fi && \
-	PACKAGED_SETTINGS=$$(echo $$PACKAGED_SETTINGS | xargs) && echo "Packaged settings: $$PACKAGED_SETTINGS"
-endef
-
```
Comment (myleshorton) [inline Makefile]: We effectively weren't using this anymore before, so this is just cleanup.

File: src/github.com/getlantern/flashlight/client/balancer.go [hunk 2]
```diff
@@ -20,31 +15,36 @@ func (client *Client) getBalancer() *balancer.Balancer {
...
+			dialers = append(dialers, dialer)
+			if dialer.QOS > highestQOS {
+				// If this dialer as a higher QOS than our current highestQOS, set it as
+				// the highestQOSFrontedDialer.
+				highestQOSFrontedDialer = fd
+				highestQOS = dialer.QOS
+			}
 		}
-	}
+	*/
```
Comment (myleshorton) [inline src/github.com/getlantern/flashlight/client/balancer.go]: Don't use fronted servers at all for now.

**Comment by xiam** [comment]
What's this `chained.yaml` file?

```
Generating distribution package for windows/386...
Build tags: prod
cat: installer-resources/chained.yaml: No such file or directory
Makefile:200: recipe for target 'docker-package-windows' failed
make: *** [docker-package-windows] Error 1
make: *** [package-windows] Error 2
```

**Comment by myleshorton** [comment]
Hey @xiam! It's actually just the file generated from that `fetchcfg.py` script. I'll add a dummy one in.

**Comment by xiam** [comment]
Hey @myleshorton, @fffw, are you planning to submit more commits to this branch? It looks good to me except for lantern-mobile, I think that's going to continue failing for a while until we merge our modifications into it.

**Comment by fffw** [comment]
@xiam Still a few changes needs to be done but it's pretty close I think.

**Comment by myleshorton** [comment]
Merging!

**Pull request merged**